language/move-lang/functional-tests/tests/diem/on_chain_config/change_script_allowlist.move

```diff
@@ -123,3 +90,51 @@ module FooConfig2 {
     }
 }
 // check: INVALID_MODULE_PUBLISHER
+
+// disable all non-admin txes
+//! new-transaction
+//! sender: diemroot
+script {
+use 0x1::DiemAccount;
+fun main(account: &signer) {
+    DiemAccount::halt_all_transactions(account);
+}
+}
+// check: "Keep(EXECUTED)"
+
+// sending fails
+//! new-transaction
+script {
+fun main() {
+}
+}
+// check: "Discard(UNKNOWN_SCRIPT)"
+
+// re-enable
+//! new-transaction
+//! sender: diemroot
+script {
+use 0x1::DiemAccount;
+fun main(account: &signer) {
+    DiemAccount::resume_transactions(account);
+}
+}
+// check: "Keep(EXECUTED)"
+
+// sending succeeds again
+//! new-transaction
+script {
+fun main() {
+}
+}
+// check: "Discard(UNKNOWN_SCRIPT)"
+
+// insufficient privs
+//! new-transaction
+script {
+use 0x1::DiemAccount;
+fun main(account: &signer) {
+    DiemAccount::halt_all_transactions(account);
+}
+}
+// check: "Keep(EXECUTED)"
```

Comment:
This doesn't seem quite right to me.  I would expect it to be an abort of some flavor since it's not being executed by DR.

Reply:
Yes, that is indeed failing :)